container-host-files/etc/scf/config/scripts/configure-az.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+if test -z "${AZ_LABEL_NAME}"
+then
+    echo "AZ: No label configured"
+    echo "AZ: Skipping"
+else
+    # AZ override processing is in effect.
+
+    # Locate the kubectl binary.
+    kubectl="/var/vcap/packages/kubectl/bin/kubectl"
+
+    # Determine the name of the kube worker node this container is
+    # executing on.
+
+    node="$($kubectl get pod $(hostname) -o jsonpath='{.spec.nodeName}')"
```

Comment:
Just realized we never quoted `$kubectl` and `$(hostname)` here…

Reply:
Done with commit 158b9b51fe310632299747b6a4e80eef87d80457